### ISSUE TITLE
Documentation Section - Profiling Website

### ISSUE THREAD

**Issue by SilasReinagel** [clinicjs/node-clinic-doctor#375]
I read through the documentation. It seems easy to run Clinic against a node app.

Is there a way to run Clinic against a statically generated React website? Or to run it against a Webpack-hosted app instead of direct node?

**Comment by RafaelGSS** [comment]
No, Clinic relies on Node.js internals. You can run it in the SSR of your React app.

**Comment by SilasReinagel** [comment]
Okay. This makes sense. I guess Clinic can work on Website with SSR/SSG, but not pure web client apps.
